.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,49 @@
+name: Run Benchmarks
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Runs daily at midnight
+  pull_request:
```

Comment:
**question:** Does this mean it doesn't run on PRs? Or that it does?
I think we want the first.

Reply:
It does run on PRs

Reply:
I am in favor of running it on PR's especially after we remove the unnecessary benches which will make it faster

Reply:
Yeah, if it doesn’t impact CI runtime too much, im happy for thus to run in every PR. Otherwise a cron seems more reasonable. 
Also, if we run on every PR and merge, we don't need to run with a cron job, right?

Reply:
I don't see cron job and PR jobs as exclusive, so I am leaning to enable benchmark for both for now and later can adjust if needed.